Unit tests

RogerAnselmo opened the pull request.

Commit:
unit-tests -> userpage-tests

Commit:
unit-tests -> BaseTest

unit-tests -> IWebElementExtensionsTest

igorquintaes commented on InstaSharp.UnitTests/Specs/Extensions/IWebElementExtensionsTest.cs:
```diff
@@ -0,0 +1,64 @@
+        public class AdjustElementExibitionTest: IWebElementExtensionsTest
+        {
+            [Test]
+            public void ShouldReturnWebElement()
```
Acredito que o nome do teste poderia ser mais específico, para garantir que não só retorne um webelement, mas especificamente o mesmo enviado como parâmetro do método.

Commit:
unit-tests -> BaseTest

changing test name from ShouldReturnWebElement to ShouldReturnTheSameWebElementPassedByParameter
